neo4j/README.md

```diff
@@ -38,26 +40,37 @@ To install the neo4j check on your host:
 
 ### Metrics
 
-See [metadata.csv][6] for a list of metrics provided by this check.
+**Neo4j Version 4**
+Neo4j 4 metrics are collected as documented [here][11]. The most commonly monitored metrics are provided in the out-of-the-box dashboards. 
+
+**Neo4j Version 5**
+Neo4j 5 metrics are collected as documented [here][10]. The most commonly monitored metrics are provided in the out-of-the-box dashboards. 
+
+Please note each version collects a different set of metrics. The versions are listed in the description of the metric.
+
+See [metadata.csv][6] for the full list of metrics provided by this check.
 
 ### Service Checks
 
 Service check `neo4j.prometheus.health` is submitted in the base check
 
 ### Events
 
-neo4j does not include any events.
+Neo4j does not include any events.
```

Comment:
Let's move this under the overview sections so it will appear on the Neo4j tile

Reply:
@davidfauth Can you remove this section since we moved it up?

Reply:
See if this works.